src/bitcoinrpcclient/bitcoinrpcclient.go

```diff
@@ -66,6 +67,11 @@ func NewBitcoinRPCClient(rpcAddress string) (*BitcoinRPCClient, error) {
 	return client, nil
 }
 
+// NewBitcoinRPCClientForIntegrationTest creates a new BitcoinRPCClient based on envvar
+func NewBitcoinRPCClientForIntegrationTest() (*BitcoinRPCClient, error) {
+	return NewBitcoinRPCClient(os.Getenv("TEST_INTEGRATION_RPC_ADDRESS"))
```

Comment:
Probably make sense to define the environment variable names as constants somewhere all in once place.

Reply:
Agreed, although I propose we do it in a separate PR

I favor putting reusable test helpers in a `/testing/` subpackage

Reply:
I started a package `/test/`